src/core/example_settings.py

```diff
@@ -1,464 +1,24 @@
-#!/usr/bin/env python -W ignore::DeprecationWarning
-__copyright__ = "Copyright 2017 Birkbeck, University of London"
-__author__ = "Martin Paul Eve & Andy Byers"
-__license__ = "AGPL v3"
-__maintainer__ = "Birkbeck Centre for Technology and Publishing"
-
-"""
-Django settings for core project.
-
-Generated by 'django-admin startproject' using Django 1.8.5.
-
-For more information on this file, see
-https://docs.djangoproject.com/en/1.8/topics/settings/
-
-For the full list of settings and their values, see
-https://docs.djangoproject.com/en/1.8/ref/settings/
-"""
-
-# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-import os
-import sys
-import logging
-
-from django.contrib import messages
-
-from core import plugin_installed_apps
-
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-PROJECT_DIR = os.path.dirname(BASE_DIR)
-sys.path.append(os.path.join(BASE_DIR, "plugins"))
-
-# Quick-start development settings - unsuitable for production
-# See https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
 # You should change this key before you go live!
 SECRET_KEY = 'uxprsdhk^gzd-r=_287byolxn)$k6tsd8_cepl^s^tms2w1qrv'
-
-# SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
-COMMAND = sys.argv[1:]
-IN_TEST_RUNNER = COMMAND[:1] == ['test']
-ALLOWED_HOSTS = ['*']
-
-ENABLE_TEXTURE = False
```

Comment:
This should be a non-global setting.

Reply:
I don't see this used anywhere, what is its purpose?

Reply:
Its used in settings.py and will be used to enable a feature that is half finished... maybe leave it for now.